### PR TITLE
git commands

### PR DESCRIPTION
**Pull request by GowthamReddy-E**
this file has some git commands .if any commands missing please add it and rise pr

### CODE REVIEW

**Comment by Mahammad046** [comment]
ok

File: k8s commands.txt
Comment (Mahammad046) [inline k8s commands.txt:1]: good job

**Review by Mahammad046 — APPROVED** [review]
good job

**Comment by Narendram22** [comment]
Good Job

**Review by Narendram22 — APPROVED** [review]
Nice

**Review by omnath1234 — APPROVED** [review]

**Comment by Mahammad046** [comment]
ok

**Comment by reddyjalla** [comment]
This one is nice

**Pull request closed without merging**